RawSpeed/DngDecoder.cpp

```diff
@@ -503,6 +504,37 @@ void DngDecoder::decodeMetaDataInternal(CameraMetaData *meta) {
   if (mRootIFD->hasEntryRecursive(ISOSPEEDRATINGS))
     mRaw->metadata.isoSpeed = mRootIFD->getEntryRecursive(ISOSPEEDRATINGS)->getInt();
 
+  // Set the make and model
+  if (!(mRootIFD->hasEntryRecursive(MAKE) && mRootIFD->hasEntryRecursive(MODEL))) {
+    if (mRootIFD->hasEntryRecursive(UNIQUECAMERAMODEL)) {
```

Comment:
This was just a fallback to always have something there in case make or model doesn't exist. I don't recall if I actually found a file with this issue, it was probably just defensive coding.